x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -228,6 +228,16 @@ All actions that are required to manage the lifecycle of a leader index, which
 includes <<ccr-post-forget-follower,forgetting a follower>>. This
 privilege is necessary only on clusters that contain leader indices.
 
+`managed_admin_ops`::
+Permits auto-creation of indices and data streams only if a template covering
+the requested name exists. An auto-create action is the result of an index or
+bulk request on a non-existent index or data stream rather than an explicit
```

Comment:
```suggestion
the requested name exists. An auto-create action is the result of an <<docs-index_,index>> or
<<docs-bulk,bulk>> request that targets a non-existent index or data stream rather than an explicit
```